awx/ui/src/screens/Application/shared/Application.helptext.js

```diff
@@ -0,0 +1,9 @@
+import { t } from '@lingui/macro';
+
+const applicationHelpTextStrings = () => ({
```

Comment:
Since the translations here are just strings, there's no need to make this an anonymous function. An object literal would be preferred.